.github/workflows/tests.yml

```diff
@@ -51,15 +51,15 @@ jobs:
           set -xeo pipefail
           if [[ "${{ runner.os }}" == "Windows" ]]; then
             echo "Setting env vars for Windows"
-            echo "AZURE_CI_WINDOWS=true" >> $GITHUB_ENV
-            echo "SOUND_CARD_BACKEND=rtmixer" >> $GITHUB_ENV
-            echo "SOUND_CARD_NAME=Speakers" >> $GITHUB_ENV
-            echo "SOUND_CARD_FS=48000" >> $GITHUB_ENV
-            echo "SOUND_CARD_API=Windows WDM-KS" >> $GITHUB_ENV
```

Comment:
@tomstoll turns out this was breaking the tests... seems like using WDM-KS is fine if you never *also* try to use something else like MME. I just switch to MME below and it's all good